.github/workflows/conflicts.yml

```diff
@@ -0,0 +1,17 @@
+name: No unresolved conflicts
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main, localization ]
+jobs:
+  detect-unresolved-conflicts:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: List files with merge conflict markers
+        run: git --no-pager grep "<<<<<<<" ":(exclude).github/" || true
```

Comment:
Praise: I love the simple logic here!

Reply:
I was almost to make it simpler with the `--quiet` parameter to `git grep`, but unfortunately that fails if it finds _no_ matches. I'm sure there's a clever Bash trick to invert the exist code, but :shrug: